modules/storage-rocksdb/src/test/java/org/apache/ignite/internal/storage/rocksdb/instance/SharedRocksDbInstanceTest.java

```diff
@@ -277,6 +281,38 @@ void testConcurrentSortedIndexReadAndCreate() {
         assertThat(getIndexFuture.join().stream().map(IndexColumnFamily::indexId).collect(toList()), contains(0));
     }
 
+    @RepeatedTest(10)
+    void testConcurrentSortedIndexDeleteAndRead() {
```

Comment:
Where is the concurrency?

Reply:
There was concurrency previously, because we removed the index from the map after the flush, not before. I used this test to reproduce the issue. Looks like this is not needed now, I've simplified the test